airtime_mvc/application/models/RabbitMq.php

```diff
@@ -89,7 +89,7 @@ public static function SendMessageToAnalyzer($tmpFilePath, $importedStorageDirec
         if (array_key_exists("dev_env", $CC_CONFIG)) {
             $devEnv = $CC_CONFIG["dev_env"];
         }
-        $config = parse_ini_file("/etc/airtime-saas/rabbitmq-analyzer-" . $devEnv . ".ini", true);
+        $config = parse_ini_file("/etc/airtime-saas/".$devEnv."/rabbitmq-analyzer.ini", true);
```

Comment:
Should we make this fallback to the production rabbitmq config if that file doesn't exist here too?